app/views/content/shared/teaching/_funding.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="funding">
+  <h3>Fund your teacher training</h3>
+  <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v :bursaries_postgraduate_chemistry %> or scholarships of <%= v :scholarships_chemistry %></a> are available for eligible trainee <%= @front_matter["subject"] %> teachers.</p>
+  <p>You can get a bursary or scholarship alongside a tuition fee and maintenance loan. Even if you've had financial support for a previous course, you could still be eligible for funding to support your teacher training.</p>
+  <p>You may also be able to get <a href="/funding-and-support">extra funding and support</a> if you're:</p>
```

Comment:
I think we need to break this partial up @Sarah-DfE @joegibb @sarahcrack - it's not going to work on other subject pages as is

I think we would need
* One partial with just the bursary/scholarship information ( to be used for the subset of subjects which have both)
* One partial with just bursary information (to be used for the subset of subjects which have just bursaries)
* One partial with finance information (to be used for all subjects)

I think some of the wording is going to need tweaking as well - you can't say `You may **also** be able to get extra funding and support` on the pages for subjects without bursaries and scholarships - it won't make sense!

Reply:
So like this?

bursary/scholarship:
Tax-free bursaries of xxxx or scholarships of xxxx are available for eligible trainee xxxx teachers.
You can get a bursary or scholarship alongside a tuition fee and maintenance loan.

bursary only:
Tax-free bursaries of xxxx are available for eligible trainee xxxx teachers.
You can get a bursary alongside a tuition fee and maintenance loan.

All:
Even if you've had financial support for a previous course, you could still be eligible for funding to support your teacher training.

You may be able to get [extra funding and support](https://getintoteaching.education.gov.uk/funding-and-support) if you're:

a parent or carer
disabled
a veteran

Reply:
Thanks both! This has been rejigged into 3 partials now:
- bursary_scholarship
- bursary_only
- funding_support

Title of 'Fund your teacher training' has been moved back into the `_article`. 
Would be great to have a check of this once I push up the changes.